pkg/logs/schedulers/ad/scheduler.go

```diff
@@ -80,7 +81,7 @@ func (s *Scheduler) Schedule(configs []integration.Config) {
 				s.mgr.AddSource(source)
 			}
 		default:
-			// invalid integration config
+			log.Debugf("Invalid integration config: %s, ignoring it", s.configName(config))
```

Comment:
I tried to stick with the format of another log above
https://github.com/DataDog/datadog-agent/blob/main/pkg/logs/schedulers/ad/scheduler.go#L73

That case seems to be only when this is false
https://github.com/DataDog/datadog-agent/blob/main/pkg/logs/schedulers/ad/scheduler.go#L127

I am not sure what this means
```
// newSources returns true if the config can be mapped to sources.
func (s *Scheduler) newSources(config integration.Config) bool {
return config.Provider != ""
```
a non null string in provider means it can be mapped to source? But source are not mandatory I think, or appear to be arbitrary. I didn't dig into that